lua/wikis/valorant/Infobox/Team/Custom.lua

```diff
@@ -52,11 +52,7 @@ end
 ---@return string?
 function CustomTeam:createBottomContent()
 	if not self.args.disbanded then
-		return Template.expandTemplate(
-			mw.getCurrentFrame(),
-			'Upcoming and ongoing tournaments of',
-			{team = self.pagename}
-		)
+		return UpcomingTournaments{name = self.pagename, onlyHighlightOnValue = 'highlighted'}
```

Comment:
onlyHighlightOnValue is redundant with how the highlightCondition looks for this wiki